extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -150,29 +152,73 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					// go back to select setting quickpick if user escapes from inputting the setting name in case they changed their mind
 					continue;
 				}
-
-				const newConnectionStringValue = await vscode.window.showInputBox(
+				let connectionStringOptions: vscode.QuickPickItem[] = [
 					{
-						title: constants.enterConnectionString,
-						ignoreFocusOut: true,
-						value: 'Server=localhost;Initial Catalog={db_name};User ID=sa;Password={your_password};Persist Security Info=False',
-						validateInput: input => input ? undefined : constants.valueMustNotBeEmpty
+						label: constants.connectionProfile
+					},
+					{
+						label: constants.userConnectionString
 					}
-				) ?? '';
-
-				if (!newConnectionStringValue) {
-					// go back to select setting quickpick if user escapes from inputting the value in case they changed their mind
-					continue;
+				];
+				const selectedConnectionString = await vscode.window.showQuickPick(connectionStringOptions, {
+					canPickMany: false,
+					title: constants.selectConnectionString,
+					ignoreFocusOut: true
+				});
+				if (!selectedConnectionString) {
+					// User cancelled
+					return;
 				}
-
-				try {
-					const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, newConnectionStringValue);
-					if (success) {
-						connectionStringSettingName = newConnectionStringSettingName;
+				if (selectedConnectionString.label === constants.userConnectionString) {
```

Comment:
Add a comment explaining that this section is for users enter connection string manually